scripts/deploy/terraform/aws-dev/main.tf

```diff
@@ -0,0 +1,227 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+resource "aws_ecs_cluster" "aws-ecs-dev-cluster" {
+  name = "${var.app_name}-${var.app_environment}-cluster"
+  tags = {
+    Name        = "${var.app_name}-ecs"
+    Environment = var.app_environment
+  }
+}
+
+resource "aws_cloudwatch_log_group" "log-group" {
+  name = "${var.app_name}-${var.app_environment}-logs"
+
+  tags = {
+    Application = var.app_name
+    Environment = var.app_environment
+  }
+}
+
+data "template_file" "env_vars" {
+  template = file("env_vars.json")
+
+  vars = {
+    aws_access_key_id     = var.AWS_ACCESS_KEY_ID
+    aws_secret_access_key = var.AWS_SECRET_ACCESS_KEY
+    lambda_func_arn       = "${aws_lambda_function.terraform_lambda_func.arn}"
+    lambda_func_name      = "${aws_lambda_function.terraform_lambda_func.function_name}"
+  }
+}
+
+resource "aws_ecs_task_definition" "manager-task" {
```

Comment:
We'll need a different entry point for manager task. You can override the command for manager task once you add the management page.